### ISSUE TITLE
Error: cb is not a function when output file set as /dev/null

### ISSUE THREAD

**Issue by mavidser** [Unitech/pm2#3037]
## What's going wrong?
Deployment is failing. Error stack:

```
TypeError: cb is not a function
    at SyncWriteStream.write (/opt/app/node_modules/pm2/lib/ProcessContainer.js:247:18)
    at Console.log (console.js:43:16)
    at Object.<anonymous> (/opt/app/core/bin/www:6:9)    # a console.log
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at /opt/app/node_modules/pm2/lib/ProcessContainer.js:324:23
    at /opt/app/node_modules/pm2/node_modules/async/dist/async.js:421:16
```

## How could we reproduce this issue?
Set either `error_file` or `out_file` in configuration as `/dev/null`. Current config:

```
{
      name      : 'Core Service',
      script    : 'core/bin/www',
      exec_mode : 'cluster',
      merge_logs: true,
      instances : 16,
      log_file  : '/dev/null',
      error_file: '/dev/null',
      out_file  : '/dev/null',
}
```

**Comment by Unitech** [comment]
Fixed on 2.6.1, can you update to latest pm2

```
$ npm install pm2@latest -g
$ pm2 update
```

**Comment by mavidser** [comment]
Upgrading to 2.6.1 does fix this particular error.

However, now, when setting `/dev/null` as the `error_file` and `out_file`, 
`pm2 logs` command doesn't echo the logs to stdout, which is a change in behaviour from 2.1.6, which did echo them.

I'm not sure if this is by design or not, but I'm having a hard time figuring out how to stream the logs to stdout without writing them to a file (which could also be helpful if you use any other utility for log management/rotation/etc). Writing to `/dev/stdout` doesn't work either.

**Comment by Unitech** [comment]
We will fix that for the next release, logs will continue to be passed to the bus

**Comment by Unitech** [comment]
Fixed on recently published pm2 (2.7.0):

```bash
$ npm install pm2@latest -g
$ pm2 update
```

Let me know if there is any issue
